runner/src/server/plugins/crumb.ts

```diff
@@ -16,12 +17,14 @@ export const configureCrumbPlugin = (
     plugin: crumb,
     options: {
       logUnauthorized: true,
-      enforce: routeConfig
-        ? routeConfig.enforceCsrf || false
+      enforce: routeConfig?.enforceCsrf
+        ? true
+        : config.isTest
+        ? false
```

Comment:
I try to avoid ternary, especially double ternaries since they get difficult to read. 
```suggestion
      enforce: routeConfig?.enforceCsrf ?? !config.isTest
```
if routeConfig.enforceCsrf is defined, then use that value, otherwise default to !config.isTest